### PR TITLE
[CLI] Add support for voting with multiple stake pools

### PR DESCRIPTION
**Pull request by movekevin**
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4853)

### CODE REVIEW

File: crates/aptos/src/governance/mod.rs [hunk 1]
```diff
@@ -226,17 +229,18 @@ pub struct SubmitVote {
 
     #[clap(flatten)]
     pub(crate) txn_options: TransactionOptions,
-    #[clap(flatten)]
-    pub(crate) pool_address_args: PoolAddressArgs,
+    /// Comma separated list of pool addresses.
+    #[clap(long)]
+    pub(crate) pool_addresses: String,
```
Comment (gregnazario) [inline crates/aptos/src/governance/mod.rs:234]: use `multiple_values=true` and then you don't have to parse commas, and add `load_account_arg` as the parser from one of the other locations.  Then you'll just get AccountAddresses really easily (space separated)

**Review by gregnazario — APPROVED** [review]
Looks fine, just let's clean that up to use the spaces instead of commas, and keep from doing extra parse logic

File: crates/aptos/src/governance/mod.rs (hunk 1 above)
Comment (movekevin) [reply crates/aptos/src/governance/mod.rs]: Updated

**Review by sherry-x — APPROVED** [review]
Amazing!!!

**Pull request merged**